src/theme/table.scss

```diff
@@ -1,3 +1,7 @@
+tr {
+  height: 50px !important;
```

Comment:
`src/components/table/VirtualScrollTable.tsx:47`
change there the `itemSize` to 50